plugins/filter/listify.py

```diff
@@ -0,0 +1,152 @@
+# Copyright: (c) 2017, Ramses Smeyers <rsmeyers@cisco.com>
```

Comment:
```suggestion
#!/usr/bin/python
# -*- coding: utf-8 -*-

# Copyright: (c) 2023, Shreyas Srish <ssrish@cisco.com>
```

Reply:
This is Ramses' code so I'll leave it the way it is :)

Reply:
I modified the code. So I included my name, just in case people have questions